ProcessMaker/processmaker
# FOUR-16819 Implement rate limit for security log of request errors

## Rate Limit to Improve Log Performance
## Solution
- Implement rate limit for security log of request errors

## How to Test
- Create a process with an script that throws an exception
- Run the process
- Security log should register only one RequestError per day per request

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-16819

## Code Review Checklist
